scripts/bash/gitlab_helpers.sh

```diff
@@ -0,0 +1,382 @@
+#!/bin/bash
+
+readonly RAILS_CMD_PATH=$(pwd)/railscmd.rb
+
+check_installation () {
+    local logName=$1
+    #In case Gitlab instance get restarted, skip script.
+    logger $logName "INFO" "Checking whether Gitlab is installed"
+    GILAB_VERSION_FILE=/opt/gitlab/version-manifest.txt
+    GITLAB_INSTALLED="false"
+    if [ -f "$GILAB_VERSION_FILE" ]; then
+        logger $logName "DEBUG" "$GILAB_VERSION_FILE exists"
+        GITLAB_INSTALLED="true"
+        GITLAB_EE_VERSION=$(grep gitlab-ee $GILAB_VERSION_FILE | cut -d " " -f2)-ee
+        logger $logName "INFO" "Installed Gitlab version is $GITLAB_EE_VERSION"
+    else         
+        logger $logName "DEBUG" "$GILAB_VERSION_FILE does not exist"
+    fi
+}
+
+
+gitlab_depsInstall () {
+    local logName=$1
+    # Install Dependencies
+    logger $logName "INFO" "Running package updater apt-get update"   
```

Comment:
Just updating packages.
If you want verbose on commands this logic needs to be in exec_wrapper

Reply:
But I don't want all my commands to be verbose

Reply:
So it need to be an option verbose debug or not.
We will use the new wrapper when it'll be public, please add a task for that.